DOCSUP-3871: Document prefer_not_to_merge option (#17090)
ClickHouse/ClickHouse: docs/en/operations/system-tables/storage_policies.md

-   `disks` ([Array(String)](../../sql-reference/data-types/array.md)) — Disk names, defined in the storage policy.
-   `max_data_part_size` ([UInt64](../../sql-reference/data-types/int-uint.md)) — Maximum size of a data part that can be stored on volume disks (0 — no limit).
-   `move_factor` ([Float64](../../sql-reference/data-types/float.md)) — Ratio of free disk space. When the ratio exceeds the value of configuration parameter, ClickHouse start to move data to the next volume in order.

> -   `prefer_not_to_merge` ([UInt8](../../sql-reference/data-types/int-uint.md)) — Setting to disable merging for volumes in multi-disk configuration. When this setting is enabled, disk merging is not allowed.

Reviewer: ```suggestion
-   `prefer_not_to_merge` ([UInt8](../../sql-reference/data-types/int-uint.md)) — Value of the `prefer_not_to_merge` setting. When this setting is enabled, disk merging is not allowed.
```
